airbyte-integrations/connectors/destination-e2e-test/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'application'
     id 'airbyte-docker'
     id 'airbyte-integration-test-java'
+    id 'airbyte-spec-acceptance-test'
```

Comment:
This and the nearly empty `acceptance-test-config.yml` is what needs to be added to each connector that doesn't run the regular source acceptance tests right now (like the destinations)